portfolio/src/main/webapp/script.js

```diff
@@ -149,23 +147,71 @@ function deleteData(data) {
 // Creates a Map with Markers of Chick fil A locations in Tennessee
 function createMap() {
     const map = new google.maps.Map(
-    document.getElementById('map'),
-    {center: {lat: 35.832308, lng: -86.317786}, zoom: 8});
+        document.getElementById('map'), {
+            center: {
+                lat: 35.832308,
+                lng: -86.317786
+            },
+            zoom: 8
+        });
 
     var iconBase = 'https://maps.google.com/mapfiles/kml/shapes/';
-    
+
     const fiskUniversityMarker = new google.maps.Marker({
-    position: {lat: 36.1663542, lng:  -86.8031514},
-    map: map,
-    title: 'My College Campus',
-    icon: iconBase + 'parking_lot_maps.png'
+        position: {
+            lat: 36.1663542,
+            lng: -86.8031514
+        },
+        map: map,
+        title: 'My College Campus',
+        icon: iconBase + 'parking_lot_maps.png'
     });
 
     fetch('/chicFilA-data').then(response => response.json()).then((chicFilACoordinates) => {
-    chicFilACoordinates.forEach((chicFilACoordinate) => {
-      new google.maps.Marker(
-          {position: {lat: chicFilACoordinate.lat, lng: chicFilACoordinate.lng}, map: map});
+        chicFilACoordinates.forEach((chicFilACoordinate) => {
+            new google.maps.Marker({
+                position: {
+                    lat: chicFilACoordinate.lat,
+                    lng: chicFilACoordinate.lng
+                },
+                map: map
+            });
+        });
+    });
+}
+
+function fetchLogin() {
```

Comment:
if I'm understanding this correctly - if I'm already logged in when i come to your website, the website will show a login button, and when i click it, it will log me out - is that correct?

Reply:
If you're already logged in, the website shows a logout button, and when you click it, it logs you out

Reply:
that only happens when the comment form is loaded... loading comments doesn't seem to have much to do with the login button? we generally prefer to group related units of functionality together in code, it's both easier for readers to understand, and leads to less bugs in the future. for example, if i was making changes to your webapp, and decided to rip out the comment functionality - all of a sudden the login part would be broken too. let's move the login button stuff out of the showCommentForm() function